src/uu/mkfifo/src/mkfifo.rs

```diff
@@ -48,7 +45,7 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
         None => {
             return Err(USimpleError::new(
                 1,
-                get_message("mkfifo-error-missing-operand"),
+                translate!("mkfifo-error-missing-operand", "util_name" => uucore::execution_phrase()),
```

Comment:
Is this intentional?
```suggestion
                translate!("mkfifo-error-missing-operand"),
```